test/decl/var/usage.swift

```diff
@@ -287,6 +287,14 @@ func sr964() {
       print(suspiciousSetter) // expected-warning {{setter argument 'newValue' was never used, but the property was accessed}} expected-note {{did you mean to use 'newValue' instead of accessing the property's current value?}} {{13-29=newValue}}
     }
   }
+  struct MemberGetter {
+    var suspiciousSetter: String {
+      get { return "" }
+      set {
+        print(suspiciousSetter) // expected-warning {{setter argument 'newValue' was never used, but the property was accessed}} expected-note {{did you mean to use 'newValue' instead of accessing the property's current value?}} {{15-31=newValue}}
+      }
+    }
+  }
```

Comment:
Can you add one for classes too? I've got extra paranoia now that we let the first time through.

Reply:
Yea, good thought - I also added an extension test. Unfortunately, I did the tests in the github editor since I don't have a working development environment right now! Friday I handed in my work laptop and wont get a new laptop till Monday. The tests were pretty easy so I expect them to pass in CI, but if there's any issues, I can pick it up early next week.